about.html

```diff
@@ -10,54 +10,113 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<header class='navbar'>
+    <img src= "lambda-black.png">
+      <nav class='nh'>
```

Comment:
I would like to see a _little_ bit better of a class name here. It's just if I looked in your css and saw you styling `nh` I would have no clue what that is